Weekly_Work/Week1/Code_changes-git/Metrics_Root_class.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import json
+import datetime
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+# In[7]:
+
+
+class Metric():
+    
+    def __init__(self, path, since=None, until=None, source_code_exclude_list=None):
```

Comment:
Please, include heading comments, so that it is more clear what the class is about, but specially all the functions. Check the heading comments in functions in Perceval to get an idea about how to write those comments.

Reply:
As mentioned above, I'll add those in the following pull request, once I have properly implemented the structure and functionality of the classes
Thanks!

Reply:
Great, thanks. Have in mind that proper comments (of course, not overcommenting) helps both the reader, to better understand the code and its rationale, but also the writer, to better structure the code. But i'm sure you already know this ;-)